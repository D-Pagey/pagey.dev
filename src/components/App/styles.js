import styled, { createGlobalStyle } from 'styled-components';
import { normalize } from 'polished';

export const GlobalStyle = createGlobalStyle`
    ${normalize()};

    body {
    margin: 0;
    font-family: 'Nunito', sans-serif;
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
  }
  
  code {
    font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
      monospace;
  }
`;

export const Wrapper = styled.div`
    margin: 0 auto;
    max-width: 900px;
    padding: 1.5rem;
`;
