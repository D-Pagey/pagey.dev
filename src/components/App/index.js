import React from 'react';
import * as S from './styles';
import CV from '../CV';

const App = () => {
  return (
    <S.Wrapper>
      <S.GlobalStyle />

      <CV />
    </S.Wrapper>
  );
};

export default App;
