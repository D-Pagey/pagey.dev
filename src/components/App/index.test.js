import React from 'react';
import App from '.';

it('should render', () => {
  const { container } = render(<App />);
  expect(container.firstChild).toMatchSnapshot();
});
