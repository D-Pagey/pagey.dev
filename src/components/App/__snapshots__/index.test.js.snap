// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render 1`] = `
.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin: 0 0 1rem 0;
}

.c2 {
  color: initial;
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  text-transform: uppercase;
}

.c3 {
  color: initial;
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  text-transform: uppercase;
}

.c8 {
  fill: pink;
}

.c8:hover {
  fill: skyblue;
}

.c4 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 2rem 0 0;
}

.c5 {
  display: grid;
  max-width: 365px;
}

.c6 {
  border-radius: 3px;
  margin: 0 auto;
  width: 325px;
}

.c7 {
  margin: 1rem 0 0;
}

.c0 {
  margin: 0 auto;
  max-width: 900px;
  padding: 1.5rem;
}

@media (min-width:600px) {
  .c5 {
    grid-column-gap: 1rem;
    grid-template-columns: 1fr 1fr;
    grid-template-rows: max-content;
    max-width: none;
  }
}

@media (min-width:600px) {
  .c6 {
    grid-row: 1 / 3;
  }
}

<div
  class="c0"
>
  <header
    class="c1"
  >
    <a
      aria-current="page"
      class="c2 active"
      href="/"
    >
      pagey
    </a>
    <a
      class="c3"
      href="/cv"
    >
      cv
    </a>
  </header>
  <div
    class="c4"
    data-testid="pageHome"
  >
    <div
      class="c5"
    >
      <img
        alt="pagey"
        class="c6"
        src="pagey.jpg"
      />
      <p
        class="c7"
      >
        Yo, my name is Dan, I'm a JavaScript Developer and I enjoy building apps to solve problems.
      </p>
      <p
        class="c7"
      >
        I have always appreciated well done data vizualisation and so I'm learning to use D3.js within React apps. Hopefully, I can build something cool and interactive. Until then, this is my playground.
      </p>
    </div>
    <div>
      <h1>
        My Gym Usage:
      </h1>
      <svg
        height="400"
        width="500"
      >
        <line
          stroke="red"
          stroke-dasharray="2"
          stroke-width="1"
          x1="50"
          x2="500"
          y1="202.436974789916"
          y2="202.436974789916"
        />
        <line
          stroke="grey"
          stroke-width="1"
          x1="50"
          x2="50"
          y1="365"
          y2="35"
        />
        <text
          text-anchor="end"
          x="45"
          y="40"
        >
          67
        </text>
        <text
          x="0"
          y="200"
        >
          Minutes
        </text>
        <text
          text-anchor="end"
          x="45"
          y="370"
        >
          50
        </text>
        <text
          x="250"
          y="400"
        >
          Index
        </text>
        <line
          stroke="grey"
          stroke-width="1"
          x1="50"
          x2="500"
          y1="365"
          y2="365"
        />
        <text
          x="440"
          y="200"
        >
          Average
        </text>
        <g>
          <rect
            class="c8"
            height="155.2941176470588"
            width="30"
            x="50"
            y="209.7058823529412"
          />
          <text
            x="58"
            y="380"
          >
            0
          </text>
        </g>
        <g>
          <rect
            class="c8"
            height="330"
            width="30"
            x="120.83333333333333"
            y="35"
          />
          <text
            x="128.83333333333331"
            y="380"
          >
            1
          </text>
        </g>
        <g>
          <rect
            class="c8"
            height="155.2941176470588"
            width="30"
            x="191.66666666666666"
            y="209.7058823529412"
          />
          <text
            x="199.66666666666666"
            y="380"
          >
            2
          </text>
        </g>
        <g>
          <rect
            class="c8"
            height="252.35294117647058"
            width="30"
            x="262.5"
            y="112.64705882352942"
          />
          <text
            x="270.5"
            y="380"
          >
            3
          </text>
        </g>
        <g>
          <rect
            class="c8"
            height="252.35294117647058"
            width="30"
            x="333.3333333333333"
            y="112.64705882352942"
          />
          <text
            x="341.3333333333333"
            y="380"
          >
            4
          </text>
        </g>
        <g>
          <rect
            class="c8"
            height="232.94117647058826"
            width="30"
            x="404.1666666666667"
            y="132.05882352941174"
          />
          <text
            x="412.1666666666667"
            y="380"
          >
            5
          </text>
        </g>
        <g>
          <rect
            class="c8"
            height="38.8235294117647"
            width="30"
            x="475"
            y="326.1764705882353"
          />
          <text
            x="483"
            y="380"
          >
            6
          </text>
        </g>
      </svg>
    </div>
  </div>
</div>
`;
